doc/installation.md

```diff
@@ -180,13 +206,15 @@ listen bigcouch-data 127.0.0.1:15984
   balance roundrobin
     server test1.cluster1.2600hz.com 192.168.1.100:5984 check
     server test2.cluster1.2600hz.com 192.168.1.101:5984 check
-    server test3.cluster1.2600hz.com 192.168.1.102:5984 check backup
+    server test3.cluster1.2600hz.com 192.168.1.102:5984 check
+    server test4.cluster1.2600hz.com 192.168.1.103:5984 check backup
 
 listen bigcouch-mgr 127.0.0.1:15986
   balance roundrobin
     server test1.cluster1.2600hz.com 192.168.1.100:5986 check
     server test2.cluster1.2600hz.com 192.168.1.101:5986 check
-    server test3.cluster1.2600hz.com 192.168.1.102:5986 check backup
+    server test3.cluster1.2600hz.com 192.168.1.102:5986 check
+    server test4.cluster1.2600hz.com 192.168.1.103:5986 check backup
```

Comment:
Should we provide two configs, one for zone 1 and one for zone 2, where zone 1's config has zone 2 servers as backups and zone 2 has zone 1's servers as backup?